tests/test_provider.py

```diff
@@ -208,14 +230,14 @@ def validate_jobs(self, jobs):
                 self.assertIn("juju_model_uuid", labels)
                 self.assertIn("juju_application", labels)
 
-            relabel_configs = job["relabel_configs"]
-            self.assertEqual(len(relabel_configs), 1)
+            # relabel_configs = job["relabel_configs"]
```

Comment:
Should we not check for presence of `relabel_configs` ?

Reply:
Yeah I guess so, in case we refactor